tests/test_cassandra.py

```diff
@@ -68,6 +70,33 @@ def test_cassandra_execute_exception(monkeypatch):
 
     monkeypatch.setattr('zmon_worker_monitor.builtins.plugins.cassandra_wrapper.Cluster', cluster)
 
+    with pytest.raises(RuntimeError):
+        cassandra = CassandraWrapper(node, keyspace)
+        res = cassandra.execute('SELECT')
+
+        assert res == result
+
+        client.connect.assert_called_once()
+        client.shutdown.assert_called_once()
```

Comment:
I think those two asserts should be moved out of the ``with`` scope. And ``assert res == result`` is never executed.